client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -342,41 +402,28 @@ export const Admin = () => {
             token={token}
           />
 
-          <div hidden={!(loadingSemester === 1)} className="">
-            <p>
-              Adding New Semester Data. This process can take up to 15 minutes.
-            </p>
-          </div>
-
-          <div hidden={!(loadingSemester === 2)} className="">
+          <div hidden={!semAdded} className="">
```

Comment:
Also, could these messages be cleaner in any way? 

IE we don't want to really ever hard-code text like this, 1 potential thing i was thinking about is

```typescript
const [updated, setUpdated] = useState<'empty', 'semester', 'professor', 'subjects', etc.>('empty');

const successMessages = {empty: '', semester: "New Semester data successfully updated.", professor: "New Professors successfully updated" }; 

<div> {messages[updated]} <div> 
```